emc-metalnx-services/src/test/java/com/emc/metalnx/services/irods/CollectionServiceImplTest.java

```diff
@@ -140,5 +141,46 @@ public void testGetSubCollectionsAndDataObjectsUnderPathThatMatchSearchTextPagin
 		Assert.assertTrue("no recs returned", actual.size() > 1);
 
 	}
+	
+	@Test
+	public void testCreateCollectionWithParent() throws Exception {
+		
+		IRODSAccount irodsAccount = testingPropertiesHelper.buildIRODSAccountFromTestProperties(testingProperties);
+
+		IRODSServicesImpl irodsServices = new IRODSServicesImpl();
+		irodsServices.setIrodsAccount(irodsAccount);
+		irodsServices.irodsAccessObjectFactory = irodsFileSystem.getIRODSAccessObjectFactory();
+
+		CollectionServiceImpl collectionService = new CollectionServiceImpl();
+		collectionService.setIrodsServices(irodsServices);
+
+		// get new collection as DataGridCollectionAndDataObject
+		String newSubCollection = "subdir1/subdir2";
+		StringBuilder parentCollectionStringBuilder = new StringBuilder();
+		parentCollectionStringBuilder.append(irodsAccount.getHomeDirectory());
+		//parentCollectionStringBuilder.append("/subdir1");
+		
+		String currentPath = irodsAccount.getHomeDirectory();
+		String fullPath = currentPath + '/' + newSubCollection;
+		
+		DataGridCollectionAndDataObject newCollection = new DataGridCollectionAndDataObject(
+				fullPath, newSubCollection, currentPath, true);
+
+		newCollection.setParentPath(currentPath);
+		newCollection.setCreatedAt(new Date());
+		newCollection.setModifiedAt(newCollection.getCreatedAt());
+		newCollection.setInheritanceOption(false);
```

Comment:
Are these necessary for creating a collection?
Does the test pass if they are removed?

Reply:
It does pass.  I think I was just using what metalnx was doing.  I removed it and tests are still passing.